services/management/dbaas/deps.go

```diff
@@ -68,6 +69,9 @@ type dbaasClient interface {
 type versionService interface {
 	// Matrix calls version service with given params and returns components matrix.
 	Matrix(ctx context.Context, params componentsParams) (*VersionServiceResponse, error)
+	// GetLatestOperatorVersion returns latest operators versions available based on given params.
+	// It also returns PMM version that is compatible with returned operators version.
```

Comment:
as far as I see it doesn't return PMM version

Reply:
removed